docs/src/main/sphinx/release/release-455.md

```diff
@@ -0,0 +1,61 @@
+# Release 455 (28 Aug 2024)
+
+## General
+
+* Add query starting time in `QueryStatistics` in all [](admin-event-listeners).
+  ({issue}`23113`)
+* {{breaking}} Add JMX metrics for the bean
+  `trino.execution.executor.timesharing:name=TimeSharingTaskExecutor` replacing
+  metrics previously found in `trino.execution.executor:name=TaskExecutor`.
+  ({issue}`22914`)
+* Fix memory tracking issue for aggregations that could cause worker crashes
+  with out-of-memory errors. ({issue}`23098`)
+* Fix loading of custom `CatalogStore` implementations. Set `catalog.store` to
+  the name of the custom `CatalogStoreFactory`. `catalog-store.name` in
+  `etc/catalog-store.properties` is no longer required. ({issue}`22554`)
+* For file-based catalog store the relevant configuration now needs to be done
```

Comment:
The wording of this item is weird. Also, this seems like a breaking change.

Reply:
The $files table return more rows now, that can be distinguished using the value of the "content" column. This is not a breaking change.

Reply:
yes, this is breaking change (for an experimental feature).

Reply:
Yeah .. but its kinda related to the earlier one .. we might want to collapse or whatever